### DISCUSSION TITLE
HM-MOD-RPI-PCB an RPI4 unter Home Assistant Addon 3.71.12.20230826 wird nicht erkannt

### DISCUSSION THREAD

**Discussion by Maximilian4711** [OpenCCU/OpenCCU#2421] [🇩🇪 Fragen&Antworten]
Ich habe erstmalig heute das Modul HM-MOD-RPI-PCB im Home Assistent installiert und keine Erfahrungen hinsichtlich vorheriger Versionen.
Modul wird nach Installation des Addons im log zwar erkannt, jedoch nicht in der RaspberryMatic (Duty Cycle Anzeige fehlt). Im Snapshot 3.71.12.20230901 dagegen wird das Modul angezeigt und kann anlernen.

Home Assistant 2023.8.4
Supervisor 2023.08.1
Operating System 10.5
Frontend 20230802.1 - latest

**Comment by jens-maus** [comment]
Das kann ich leider nicht reproduzieren. Hier funktioniert das in Testinstallationen tadellos. Es muss da also noch irgendein anders gelagertes Problem bei deiner Installation geben die verhindert das das HM-MOD-RPI-PCB zwar korrekt beim booten erkannt wird, aber die DutyCycle anzeige fehlt.
